src/pages/Register.test.ts

```diff
@@ -67,4 +79,46 @@ describe('Register', () => {
     expect(registerButton).not.toHaveAttribute('disabled');
     expect(await axe(container)).toHaveNoViolations();
   });
+
+  it('fires off a request to Orch to register a user', async () => {
+    // Arrange
+    const user = userEvent.setup();
+    const profileSetFunction = jest.fn().mockReturnValue({});
+
+    asMockedFn(Ajax).mockImplementation(
+      () =>
+        ({
+          Metrics: { captureEvent: jest.fn() } as Partial<ReturnType<typeof Ajax>['Metrics']>,
+          User: {
+            profile: {
+              set: profileSetFunction,
+              get: jest.fn().mockReturnValue({}),
+            } as Partial<ReturnType<typeof User>['profile']>,
+          } as Partial<ReturnType<typeof Ajax>['User']>,
+        } as ReturnType<typeof Ajax>)
+    );
+
+    // Act
+    render(h(Register));
+    await user.type(screen.getByLabelText(/First Name/), 'Test Name');
+    await user.type(screen.getByLabelText(/Last Name/), 'Test Last Name');
+    await user.type(screen.getByLabelText(/Contact Email for Notifications/), 'testemail@noreply.com');
+    await user.type(screen.getByLabelText(/Organization/), 'Test Organization');
+    await user.type(screen.getByLabelText(/Department/), 'Test Department');
+    await user.type(screen.getByLabelText(/Title/), 'Test Title');
```

Comment:
UserEvent's `type` adds delay between each key to simulate a user typing.

We've found it tends to make tests painfully slow. It may be worth considering [`fireEvent.change`](https://testing-library.com/docs/dom-testing-library/api-events) instead.

Reply:
thanks for the tip!

Reply:
The tradeoff is that UserEvent handles `act` and `async` operations resulting from interactions for you.

Reply:
Given that these aren't doing any async updates, that's not a problem!